### PR TITLE
Ref: Create configuration for each service

### PR DESCRIPTION
**Pull request by HaroldCuico**
Created a configuration file for each http address for each service

### CODE REVIEW

**Review by JMark-FS — APPROVED** [review]
Looks good

File: WebApp/backend/QuizMaster.API.Gatewway/Controllers/AccountGatewayController.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Gatewway/Controllers/AccountGatewayController.cs:1]: this is very good, in preparation for prod

File: WebApp/backend/QuizMaster.API.Gatewway/Controllers/MediaGatewayController.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Gatewway/Controllers/MediaGatewayController.cs:1]: good

File: WebApp/backend/QuizMaster.API.Gatewway/Controllers/QuizCategoryGatewayController.cs
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Gatewway/Controllers/QuizCategoryGatewayController.cs:1]: good

File: WebApp/backend/QuizMaster.API.Gatewway/appsettings.json
Comment (jaymar921) [inline WebApp/backend/QuizMaster.API.Gatewway/appsettings.json:1]: very good

**Review by jaymar921 — APPROVED** [review]
approved

**Pull request merged**